website/repairnator-mongo-rest-api/yarn.lock

```diff
@@ -5,46 +5,55 @@
 abbrev@1, abbrev@1.0.x:
```

Comment:
I assume this file is autogenerated?

Reply:
Yes this file is autogenerated by yarn from package.json.

Reply:
I assume this depends on the version dependencies but I don't have a clue of what yarn is doing in that file.

Reply:
Okay. Then it should be fine, was curious about the addition the checksums as well as the many duplicates.